### PR TITLE
Release 33.4 cherrypick file handler leak

### PR DESCRIPTION
**Pull request by DaveShuckerow**

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <vendor url="https://github.com/flutter/flutter-intellij">flutter.io</vendor>
 
   <category>Custom Languages</category>
-  
+  <version>33.4.1</version>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: I would remove this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## 33.4
+ - fix file handlers leaking
```
Comment (devoncarew) [inline CHANGELOG.md]: `fix an issue with file handles leaking`?

Comment (DaveShuckerow) [reply CHANGELOG.md]: Done.

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (DaveShuckerow) [reply resources/META-INF/plugin.xml]: Done.

Comment (DaveShuckerow) [reply resources/META-INF/plugin.xml]: Just a note, this was added by `plugin generate`.

**Pull request merged**